hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/infoschema/SqlInfoSchemaTest.java

```diff
@@ -112,7 +111,7 @@ public void test_tables() {
     @Test
     public void test_datalinks() {
         // given
-        String type = DataLinkTestUtil.DummyDataLink.class.getName();
+        String type = "DUMMY";
```

Comment:
will this work for `setClassName`?

Reply:
Yes, that's the point :) It's not obvious from the method name, but actual implementation allows it.